source/opt/wrap_opkill.cpp

```diff
@@ -70,6 +73,7 @@ bool WrapOpKill::ReplaceWithFunctionCall(Instruction* inst) {
     if (undef == nullptr) {
       return false;
     }
+    undef->UpdateDebugInfo(inst);
```

Comment:
This should not been needed.  The undef and the return that follow are compiler generated instructions.  They do not represent code in the HLSL (or whatever the original source was).  They are not in any scope, and cannot actaully be reached.

Reply:
removed